src/metrics/include/metrics/v2/event.hpp

```diff
@@ -98,23 +98,28 @@ class StreamInterfaceImpl {
     StreamInterfaceImpl(const std::string& name, const ActorId& actorId)
         : _name{name},
           _actorId{actorId},
+          _in_flight{true},
           _options{},
           _response{},
           _context{},
-          _stream{_stub->StreamEvents(&_context, &_response)} {
+          _cq{},
+          _tag{(void*)1},
```

Comment:
this seems archaic - can you document why this value?

Reply:
Yeah I think I updated with a comment right as you wrote this haha. It's basically a meaningless tag associated with each call to identify them, but is still required syntactically. Maybe useful in situations where we don't know which message is which? Which isn't here.

Reply:
lol - I think some variation on what you just said would be helpful here - I'm sure I'll forget what this is in approximately 5 minutes 😸 

```
// This is used by the gRPC client to indicate which message is being completed in calls to XYZ. We only ever have 1 message in flight at a time, so the value doesn't really matter but it is required by the API.
```

I'm also curious if we should fail if the recieved tag isn't `==(void*)1`? Would that indicate that there's murder afoot? If unsure let's add a comment **or** (my preference) barf and we can remove it if/when we actually hits us.

Reply:
Good point, will comment. We actually are checking that they're equal, that's what the `gotTag == _grpcTag` part of the `finishCall()` private method does. I'll comment to clarify that though since it's not obvious without digging that the pointer passed to `_cq.Next` gets modified with the return

Reply:
Didn't want to throw in `finishCall` itself so we could use it in the destructor. IMHO though that sort of C-style pointer manipulation feels really out of place in a C++ context.